src/database/database.py

```diff
@@ -65,6 +65,12 @@ def get_user(user_id, client_id, product_id) -> GPTDockUserData:
         return _init_user(user_id, client_id, product_id)
 
 
+def get_users_count()-> int:
```

Comment:
Not a block for approval.
It could be expensive when there are a big number of users.